src/sql/queries.py

```diff
@@ -1248,6 +1248,23 @@ def round_rng(rng):
         title = f"Daily Completion - {calendar.month_name[month]} {year}"
         range_arg = "DATE_PART('day', approved_date)"
         prefix = ""
+    elif type == "objects":
```

Comment:
Note: I didn't really understand what the `di` thing was about so I didn't touch it, however I noticed in some cases we remove `-modded` from it so idk if this applies here

Reply:
Haven't looked at this in a while, but i believe the `-modded` key gets deleted for the queries where we wanna make sure it's using the nomod values, like for the stars or length. However its not being removed for the CS,OD or AR so idk